frontend/utilities/software_install_scripts.ts

```diff
@@ -0,0 +1,50 @@
+import { ISoftwareInstallerType } from "interfaces/software_installers";
+
+// @ts-ignore
+import installPkg from "../../pkg/file/scripts/install_pkg.sh";
+// @ts-ignore
+import installMsi from "../../pkg/file/scripts/install_msi.ps1";
+// @ts-ignore
+import installExe from "../../pkg/file/scripts/install_exe.ps1";
+// @ts-ignore
+import installDeb from "../../pkg/file/scripts/install_deb.sh";
```

Comment:
@ghernandez345 @gillespi314  as you can see in the scripts below the JSON file was a no-go for maintainability. What do you think about this? required a tiny modification to the webpack config but works fine (I tried doing a `console.log` of the contents)